packages/legend-extension-dsl-diagram/src/graphManager/protocol/pure/v1/transformation/pureGraph/V1_DSL_Diagram_GraphBuilderHelper.ts

```diff
@@ -92,6 +95,19 @@ export const V1_buildClassView = (
   return view;
 };
 
+const _relationshipView_simplifyPathWithErrorHandling = (
```

Comment:
I think the thing we should say here is this is an optimization we do when building the diagram; not something that will render the diagram non-compilable, so we shouldn't break.